tests/tests.py

```diff
@@ -564,7 +592,7 @@ def test_process_with_vpc_config(self):
         wanted = ["dbhost", "dbport"]
         output_names = cfn_template['Outputs'].keys()
         output_names.sort()
-        compare(output_names, wanted)
+        compare(wanted, output_names)
```

Comment:
Not proper so much as consistent, mosts of the tests run an expected/actual comparison order, best to keep them all the same if possible